src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-type Props = {
-  movie: Movie;
-};
+interface MovieProps {
```

Comment:
[ComponentName]Props
```suggestion
interface MovieCardProps {
```